LogcatCoreLib/src/main/java/info/hannes/logcat/base/LogBaseFragment.kt

```diff
@@ -26,7 +27,7 @@ import java.io.File
 abstract class LogBaseFragment : Fragment() {
 
     private var verboseItem: MenuItem? = null
-    private lateinit var logsRecycler: RecyclerView
+    private var logsRecycler: RecyclerView? = null
```

Comment:
Please avoid this null ability here and switch back to `lateinit`

Reply:
I've add a commit that reverts back to lateinit

Reply:
Great, but ...

I know this. There comes a requested change, I do it quickly and introduce an error. 

<img width="379" alt="image" src="https://user-images.githubusercontent.com/3314607/116771579-fb94c700-aa4c-11eb-8087-70d08731fdc0.png">

This doesn't work and introduce an error. The solution is to revert this line complete

Reply:
Sorry, I pushed this last change too quickly, I've pushed another commit to fix this. I'm in the middle of moving...